src/utils/index.ts

```diff
@@ -1,4 +1,4 @@
-import { toastController } from '@ionic/vue';
+import { toastController, loadingController } from '@ionic/vue';
```

Comment:
I think `loadingController` is not needed.

Reply:
Done